src/lib/machines/probeHybridisation/probeHybridisationMachine.ts

```diff
@@ -181,7 +182,7 @@ export const probeHybridisationMachine = createMachine<ProbeHybridisationContext
           if (!ctx.probeLabware.some((plw) => plw.barcode === lw.barcode)) {
             ctx.probeLabware.push({
               barcode: lw.barcode,
-              probes: [{ name: '', lot: '', plex: -1 }],
+              probes: [{ name: '', lot: '', plex: -1, costing: SlideCosting.Sgp }],
```

Comment:
I think costing has to be an empty string as initial value

Reply:
This is due to the type restriction,  I can not set the costing the empty/null as it is of tyoe SlideCosting, on the ui when the user adds a new probe row, the select box is set to empty